suite-common/secure-storage/src/secureStorage.ts

```diff
@@ -0,0 +1,38 @@
+import { Branded, BrandedArity2, Result } from '@trezor/type-utils';
+
+export type EncryptableBranded = string & Branded<string>;
+
+export type EncryptedHex<T extends EncryptableBranded> = string & BrandedArity2<'EncryptedHex', T>;
+export const asEncryptedHex = <T extends EncryptableBranded>(id: string) =>
+    id as unknown as EncryptedHex<T>;
+
+export type EncryptionUnavailable = {
+    type: 'EncryptionUnavailable';
+    message: string;
+};
+
+export const EncryptionUnavailable = (message: string) => ({
```

Comment:
why these factories named exactly the same as `type`? I don't like overlapping identifiers for types and runtime values, is that some "standard" pattern in this `ok | err` system?
If not, pls rename to `encryptionUnavailableFactory` or `createEncryptionUnavailable`
same with `DecryptionFailed`

Also pls declare the return type, currently:

```suggestion
export const EncryptionUnavailable = (message: string): EncryptionUnavailable => ({
```